index.js

```diff
@@ -38,6 +40,12 @@ const locationFormCSSPartial = fs.readFileSync(`${__dirname}/mustache-templates/
 // Start Express App
 const app = express()
 
+// Start Sentry
+Sentry.init({
+  dsn: helpers.getEnvVar('SENTRY_DSN'),
+  integrations: [new Tracing.Integrations.Postgres(), new Sentry.Integrations.Http({ tracing: true }), new Tracing.Integrations.Express({ app })],
+})
+
```

Comment:
Can this be done in Brave Alert Lib? Maybe a new Sentry module that is exported in `index.js` that is then used here by calling a `sentryHelper.setup()` or something.